lib/src/code_field/code_field.dart

```diff
@@ -1,6 +1,9 @@
 import 'dart:async';
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:js' as js;
```

Comment:
Use conditional import.

Make file `disable_spell_check_if_web.dart` with a function `disableSpellCheckIfWeb()` and `bool _isTimerSet;`.
Call it in `CodeField`'s `initState()`.